### PR TITLE
Add form validation and matching error fields to satisfy part of issue #746

### PR DESCRIPTION
**Pull request by grbarker**
This PR addresses part of issue #746. It adds form validation to the component and validation error alerts that match the corresponding field.

### CODE REVIEW

File: src/frontend/src/components/AddUser/index.tsx [hunk 1]
```diff
@@ -46,18 +82,56 @@ class AddUser extends React.Component<Props, State> {
     event.preventDefault();
 
     // Standardize email-lowerCase and trim both form inputs
+        // Standardize name-lowerCase
     this.setState(
       {
+        name: this.state.name.toLowerCase().trim(),
```
Comment (wittejm) [inline src/frontend/src/components/AddUser/index.tsx]: We should leave the `name` field as case-sensitive, so this `toLowerCase()` can be removed.

Comment (grbarker) [reply src/frontend/src/components/AddUser/index.tsx]: Ok I blew away the master adn synced with upstream. I made new branch, added the changes, and pushed it to my repo.  I'll make a pr from there. Should I close this pr?

Comment (grbarker) [reply src/frontend/src/components/AddUser/index.tsx]: And thanks for the help

Comment (wittejm) [reply src/frontend/src/components/AddUser/index.tsx]: Yep, you can close this PR.

**Pull request closed without merging**